gradlew.bat

```diff
@@ -60,11 +59,6 @@ set _SKIP=2
 if "x%~1" == "x" goto execute
 
 set CMD_LINE_ARGS=%*
-goto execute
-
-:4NT_args
-@rem Get arguments from the 4NT Shell from JP Software
-set CMD_LINE_ARGS=%$
 
 :execute
 @rem Setup the command line

```

Comment:
I suspected as much. But there are other "direct" modifications to this
file in previous accepted PRs, so I wasn't sure...

Should I do a new pull request without including it?


On Wed, Jun 8, 2016 at 1:25 PM, Marco Vermeulen <notifications@github.com>
wrote:

> In gradlew.bat
> <https://github.com/gradle/gradle/pull/657#discussion_r66289384>:
>
> > @@ -60,11 +59,6 @@ set _SKIP=2
> >  if "x%~1" == "x" goto execute
> >
> >  set CMD_LINE_ARGS=%*
> > -goto execute
> > -
> > -:4NT_args
> > -@rem Get arguments from the 4NT Shell from JP Software
> > -set CMD_LINE_ARGS=%$
> >
> >  :execute
> >  @rem Setup the command line
>
> I don't think this file is required to change as it's generated from the
> windowsStartScript.txt when we do our releases.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gradle/gradle/pull/657/files/ef11d639a372b303e0fabef3eabcdab2e39e65f0#r66289384>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAVz6kh0B_xn0d0M_E3HDIjOmdzkpFZNks5qJuzsgaJpZM4IwmuZ>
> .
>